parachain/scripts/make-snowbridge-image.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o errexit
+
+push=false
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+    -p|--push)
+        push=true
+        shift
+        ;;
+    *)
+        echo "error: unknown parameter"
+        exit 1
+        ;;
+  esac
+done
+
+version=$(git describe --tags $(git rev-list --tags --max-count=1) | sed s/v//)
+revision=$(git rev-parse HEAD)
+
+c=$(buildah from ubuntu:20.04)
```

Comment:
is buildah nativey available on ubuntu AND macos? do we need to add it to installation requirements in README?

Reply:
or is this script not expected to be used locally?

Reply:
also which runtime will this image be built for?